lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -369,6 +369,8 @@ class TypeAnalyzer : public ForwardDataFlowAnalysis<ValueKnowledge> {
       return visitConstantTensorAllocOp<AtenZerosOp>(zeros);
     } else if (auto ones = dyn_cast<AtenOnesOp>(op)) {
       return visitConstantTensorAllocOp<AtenOnesOp>(ones);
+    } else if (auto ones = dyn_cast<AtenZerosOp>(op)) {
+      return visitConstantTensorAllocOp<AtenZerosOp>(ones);
```

Comment:
The call is already at line 369, no need to redefine.

Reply:
I think I added it by mistake. Thanks!